elasticsearch-model/spec/elasticsearch/model/module_spec.rb

```diff
@@ -48,6 +48,7 @@ def self.search(query, options={})
       end
 
       DummyIncludingModel.__send__ :include, Elasticsearch::Model
+      DummyIncludingModelWithSearchMethodDefined.__send__ :include, Elasticsearch::Model
```

Comment:
Ah so there was a test for this (line 71) but it wasn't actually set up correctly. Nice!